public/js/thirdpartyservices.js

```diff
@@ -101,6 +101,14 @@ var thirdPartyServicesJsonArray = [
     link: {
       privacy: 'https://privacy.oath.com',
       terms: 'https://developer.yahoo.com/flurry/legal-privacy/terms-service/flurry-analytics-terms-service.html'
+    },
+    {
+    name: 'ACRA',
+    model: 'ACRA',
```

Comment:
Would prefer to have this as small caps i.e as 'acra' as model value

Reply:
Done 👌